src/main/java/net/dv8tion/jda/internal/requests/restaction/WebhookMessageActionImpl.java

```diff
@@ -149,7 +149,7 @@ public WebhookMessageActionImpl(JDA api, MessageChannel channel, Route.CompiledR
         Checks.notNull(data, "Data");
         Checks.notNull(options, "AttachmentOption");
         // Yes < 10 not <= 10 since we add one after this
-        Checks.check(files.size() < 10, "Cannot have more than 10 files in a message!");
+        Checks.check(files.size() < Message.MAX_EMBED_COUNT, "Cannot have more than %d files in a message!", Message.MAX_EMBED_COUNT);
```

Comment:
This pertains to attachments, not embeds.

Reply:
I'm not stupid